### PR TITLE
Implement deposit and transfer flows

### PR DESCRIPTION
**Pull request by ptare**
Issue #1040

Signed-off-by: Prasad Tare <prasad.tare@gmail.com>

### CODE REVIEW

File: core/scripts/cli/sponsor/redeem.js [hunk 1]
```diff
@@ -12,15 +12,7 @@ const redeem = async (web3, artifacts, emp) => {
   const token = await SyntheticToken.at(tokenAddress);
   const walletTokens = await token.balanceOf(sponsorAddress);
 
-  console.log("You have:");
-  console.log(
-    "Position: " +
-      fromWei(collateral) +
-      " WETH backing " +
```
Comment (nicholaspai) [inline core/scripts/cli/sponsor/redeem.js]: Why remove the messaging about collateral? I thought it was helpful to know how much collateral I had

Comment (nicholaspai) [reply core/scripts/cli/sponsor/redeem.js]: AH I see you show it in showMarketDetails

File: core/scripts/cli/sponsor/redeem.js [hunk 2]
```diff
@@ -12,15 +12,7 @@ const redeem = async (web3, artifacts, emp) => {
...
-      fromWei(position.tokensOutstanding.toString()) +
-      " synthetic tokens"
-  );
-  console.log("Wallet: " + fromWei(walletTokens) + " synthetic tokens");
+  console.log("Your wallet has: " + fromWei(walletTokens) + " synthetic tokens");
```
Comment (nicholaspai) [inline core/scripts/cli/sponsor/redeem.js]: Why show this again? Just curious

File: core/scripts/cli/sponsor/transfer.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      "Target address already has",
+      fromWei(targetCollateral),
+      "WETH. Can only transfer to an owner without a position"
+    );
+    return;
+  }
+
+  const confirmation = await inquirer.prompt({
+    type: "confirm",
+    message: "Transferring to " + input["address"] + ". This cannot be reversed!",
```
Comment (mrice32) [inline core/scripts/cli/sponsor/transfer.js]: +1 - I like the "cannot be reversed language.

File: core/scripts/cli/sponsor/redeem.js (hunk 2 above)
Comment (ptare) [reply core/scripts/cli/sponsor/redeem.js]: I don't we've shown this information anywhere, yet. This value isn't the number of tokens in the Position, it's the number of tokens in the sponsor's wallet, which they could potentially redeem.

File: core/scripts/cli/sponsor/redeem.js (hunk 1 above)
Comment (ptare) [reply core/scripts/cli/sponsor/redeem.js]: Yep, just moved it all to one place.

**Pull request merged**